odoo/netsvc.py

```diff
@@ -185,6 +185,7 @@ def is_a_tty(stream):
 PSEUDOCONFIG_MAPPER = {
     'debug_rpc_answer': ['odoo:DEBUG', 'odoo.sql_db:INFO', 'odoo.http.rpc:DEBUG'],
     'debug_rpc': ['odoo:DEBUG', 'odoo.sql_db:INFO', 'odoo.http.rpc.request:DEBUG'],
+    'log_rpc': ['odoo:INFO', 'odoo.sql_db:INFO', 'odoo.http.rpc.request:DEBUG',  'odoo.http.rpc.response:DEBUG'],
```

Comment:
Q: Is this needed over just using `debug_rpc_answer`?
nit: May be worth calling it `debug_rpc_request_response`

Reply:
Not that it was needed but added while doing the cross references in simulation and needed to have more logs.
Suggested from Miquel the name and have been using that name on log-level. When asked about adding this in odoo or not , he told wont harm to have it in acse we will resuse rather than having uncommited in some development environments.

Reply:
The first `debug` usually refers to odoo debug level, which we don't do, so probably I would go for `rpc_debug_request_response` since we are only interested in debug level for those two.